app/server.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express') // importing express library
+const server = express() // initialising express library
+const bodyParser = require('body-parser') // initialise body-parser
```

Comment:
I normally add [Helmet JS](https://www.npmjs.com/package/helmet) to my express apps for a bit of peace of mind